libretroshare/src/rsserver/rsinit.cc

```diff
@@ -1277,6 +1248,14 @@ int RsServer::StartupRetroShare()
 	//
 	mPluginsManager->loadPlugins(programatically_inserted_plugins) ;
 
+#ifdef RS_JSONAPI
+	{
+		mConfigMgr->addConfiguration("jsonApi.cfg", jsonApiServer);
+		RsFileHash dummyHash;
+		jsonApiServer->loadConfiguration(dummyHash);
```

Comment:
is this explicit loading needed?

Reply:
Don't know didn't tested if configuration (extra authentication tokens) is loaded without that, I see other services upper in the code do the same, but if you want to test you are welcome

Reply:
`mConfigMgr->loadConfiguration();`
takes care of loading all configs